app/controllers/recipe_revisions_controller.rb

```diff
@@ -4,27 +4,28 @@ class RecipeRevisionsController < ApplicationController
 
   def index
     @user             = @recipe.user
-    @revisions        = RecipeRevision.where(:recipe_id => @recipe.id)
+    @revisions        = RecipeRevision.where(recipe_id: @recipe.id)
     @recipe_permalink = @recipe.slug
   end
 
   def show
-    revision = RecipeRevision.find(params[:id])
+    @revision = RecipeRevision.find(params[:id])
 
-    if revision.revision > 1
-      previous_revision = RecipeRevision.where(:recipe_id => revision.recipe_id,
-                                               :revision => revision.revision - 1).first
-
-      @diff = Differ.diff_by_line(revision.body, previous_revision.body).format_as(:html).html_safe
+    if params[:target_id].present?
```

Comment:
I think there's actually a bit of a security vulnerability here. This exposes an interface for a user to compare their current recipe with any other one on the site. Right now that's not such a big deal since all the recipes are public, but if we go down the path of having private recipes it would need to change. So I think we should probably just force this to be scoped to the current recipe and it's forks right now.